packages/web-components/fast-ssr/src/element-renderer/fast-element-renderer.ts

```diff
@@ -175,8 +175,14 @@ function* renderAttributesSync(this: FASTElementRenderer): IterableIterator<stri
         ) {
             if (value === "" || value === undefined || value === null) {
                 yield ` ${name}`;
-            } else {
+            } else if (typeof value === "string") {
                 yield ` ${name}="${escapeHtml(value)}"`;
+            } else if (typeof value === "boolean") {
+                yield ` ${name}="${(value as any).toString()}"`;
```

Comment:
```suggestion
                yield ` ${name}="${value.toString()}"`;
```

You shouldn't need to assert value as any because the code just verified it's a boolean

Reply:
TypeScript complains `Property 'toString' does not exist on type 'never'` for it. I'll update it to `value ? "true" : "false"`